src/main/java/net/dries007/tfc/common/recipes/FoodCombiningCraftingRecipe.java

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
Note: feel free to apply all of the changes requested on this class, also to `CastingCraftingRecipe`, because it has the same issues / possible improvements to be made.